upup/pkg/fi/nodeup/command.go

```diff
@@ -760,20 +762,34 @@ func getNodeConfigFromServer(ctx context.Context, bootConfig *nodeup.BootConfig,
 		Resolver:      resolver,
 	}
 
-	u, err := url.Parse(bootConfig.ConfigServer.Server)
-	if err != nil {
-		return nil, fmt.Errorf("unable to parse configuration server url %q: %w", bootConfig.ConfigServer.Server, err)
-	}
-	client.BaseURL = *u
-	client.CAs = []byte(bootConfig.ConfigServer.CACertificates)
+	return loopThroughConfigservers(ctx, client, bootConfig.ConfigServers)
```

Comment:
I don't think an extra function is needed for now.

Reply:
Concur